.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: python
+python:
+  - 2.7
+before_install:
+  - export DISPLAY=:99
+  - sh -e /etc/init.d/xvfb start
```

Comment:
This isn't really used yet, since there aren't any integration tests.  We can leave this here, since we will have to add integration tests anyway.  They will be a bit tricky to implement, but it won't be too bad.